Using-the-Compiler-API.md

```diff
@@ -400,89 +473,9 @@ const currentDirectoryFiles = fs
 watch(currentDirectoryFiles, { module: ts.ModuleKind.CommonJS });
 ```
 
-## Transpiling a single file
-
-Currently TypeScript exposes two functions for this purpose: `transpileModule` and `transpile` (**which is deprecated**).
-Note that regardless of the name, **each one assumes that the input file is a module**.
-
-Here is the relevant signature of `transpileModule`:
```

Comment:
This was really helpful when I first visited this page. Unfortunately could not find it anymore today. Could you please add this section with the interfaces again?

Reply:
Those interfaces are in the d.ts file, and accurate there - they've changed since these were added to this doc 4-5 years ago